packages/client-config/src/client-config-writer/client_config_to_mobile_legacy_converter.ts

```diff
@@ -80,7 +82,7 @@ export class ClientConfigMobileConverter {
           Scopes: clientConfig.oauth.scope,
           SignInRedirectURI: clientConfig.oauth.redirectSignIn,
           SignOutRedirectURI: clientConfig.oauth.redirectSignOut,
-          AppClientId: clientConfig.oauth.clientId,
+          AppClientId: clientConfig.aws_user_pools_web_client_id,
```

Comment:
I'm curious about this change, was it broken?

Reply:
Yea, `clientConfig.oauth.clientId` is `undefined`. We use `clientConfig.aws_user_pools_web_client_id` for all the `AppCliendId` in the converter.

Reply:
To be safe, I just added `clientId` back 7279f13fe7a398fdb83cc258e4c355b0e04d04a9